src/recipe.module.css

```diff
@@ -0,0 +1,17 @@
+.recipe {
+  border-radius: 10px;
+  box-shadow: 0px 5px 20px rgb(71, 71, 71);
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  background: white;
+  align-items: center;
+  min-width: 40%;
+}
+
+.image {
```

Comment:
for styling purposes, would recommend that your image have some sort of padding so that it's offset from the screen edges